### ISSUE TITLE
symmetry-adapted mode with frozen window

### ISSUE THREAD

**Issue by stepan-tsirkin** [wannier-developers/wannier90#298]
Dear All,

A line in file disentangle.F90 states: 

`     if (lsitesymmetry) call io_error('in symmetry-adapted mode, frozen window not implemented yet') !YN: RS:`

I wonder if there is a crucial problem in implementing frozen window for symmetry-adapted mode, or just nobody had enough time/motivation to do it yet. Can anyone give a hint?

Regards,
Stepan.

**Comment by sfragkos** [comment]
According to the User Guide, the symmetry-adapted mode cannot be used in conjunction with the inner (frozen) energy window. Maybe there is no need for a frozen window in this type of calculations

**Comment by giovannipizzi** [comment]
Maybe @yusukenomura can give some insight - possibly there is not fundamental problem?

**Comment by stepan-tsirkin** [comment]
Yes, I discussed that with @yusukenomura  we agreed that there is no fundamental problem, and I have a general Idea how to implement it. But it is still in my todo list...
